format-all.el

```diff
@@ -737,6 +744,7 @@ unofficial languages IDs are prefixed with \"_\"."
       (and (equal major-mode 'gleam-mode) "_Gleam")
       (and (equal major-mode 'ledger-mode) "_Ledger")
       (and (equal major-mode 'snakemake-mode) "_Snakemake")
+      (and (equal major-mode 'reason-mode) "ReasonML")
```

Comment:
This line is no longer needed now that the language-id package covers Reason.

Reply:
Done